defaultRules.js

```diff
@@ -27,7 +27,10 @@ const defaultRules = {
       return false;
     }
     return true;
-  }
+  },
+  equalPassword(dataToCompare, value) {
+    return dataToCompare == value;
```

Comment:
would be better no ?

```suggestion
    return dataToCompare === value;
```

Reply:
Yes I would be better right. I Commited your comment